detect/errorloaderdetector.py

```diff
@@ -0,0 +1,33 @@
+import pandas as pd
+
+from .detector import Detector
+
+
+class ErrorsLoaderDetector(Detector):
+    """
+    Detector that loads a table of constant errors from csv.
```

Comment:
Add a comment about the expected format of the csv file.

Reply:
Done.

Reply:
Done.